docker-compose.mender-gateway.commercial.yml

```diff
@@ -1,15 +1,16 @@
 services:
-
-    #
-    # mender-gateway
-    #
-    mender-gateway:
-        image: registry.mender.io/mendersoftware/mender-gateway-qemu-commercial:mender-master
-        networks:
-            - mender
-        stdin_open: true
-        tty: true
-        privileged: true
-        environment:
-            - SERVER_URL=$SERVER_URL
-            - TENANT_TOKEN=$TENANT_TOKEN
+  #
+  # mender-gateway
+  #
+  mender-gateway:
+    image: registry.mender.io/mendersoftware/mender-gateway-qemu-commercial:mender-master
```

Comment:
Here also `${MENDER_CLIENT_TAG}`

The implementation detail is that `MENDER_CLIENT_TAG` is not really the client, but rather the "test device" or "virtual device" which contains client, addons, possibly gateway, possibly being rofs, etc